test/config/pebble-config-external-account-bindings.json

```diff
@@ -7,6 +7,10 @@
     "httpPort": 5002,
     "tlsPort": 5001,
     "ocspResponderURL": "",
+    "retryAfter": {
+        "authz": 3,
+        "order": 5
+    }
```

Comment:
I am a little worried about how much these defaults will slow down CI pipelines, especially for `authz`. Don't have a great answer about what to do, though.